src/uShip.Logging/LogBuilders/FluentLogDataBuilder.cs

```diff
@@ -82,6 +84,104 @@ public IFluentLoggerWriter Exception(Exception exception)
                 return this;
             }
 
+            /// <summary>
+            /// Reflectively add non-complex properties (exceptions like DateTime and DateTimeOffset) from an object to the Data dictionary to be included in the logger's message.
+            /// </summary>
+            /// <param name="prefix">Prefix that will be prepended to the property key names in the data dictionary.</param>
```

Comment:
An example here of how the prefix relates to the final name in the log would be nice.  E.g., given the prefix "foo", the value of `value.Bar` would be added with key "foo.Bar".